core/domain/exp_services.py

```diff
@@ -768,18 +982,25 @@ def update_exploration(
     """Update an exploration. Commits changes.
 
     Args:
-    - committer_id: str. The id of the user who is performing the update
-        action.
-    - exploration_id: str. The exploration id.
-    - change_list: list of dicts, each representing a _Change object. These
-        changes are applied in sequence to produce the resulting exploration.
-    - commit_message: str or None. A description of changes made to the state.
-        For published explorations, this must be present; for unpublished
-        explorations, it should be equal to None. For suggestions that are
-        being accepted, and only for such commits, it should start with
-        feconf.COMMIT_MESSAGE_ACCEPTED_SUGGESTION_PREFIX.
-    - is_suggestion: bool. whether the update is due to a suggestion being
-        accepted.
+        committer_id: str. The id of the user who is performing the update
+            action.
+        exploration_id: str. The id of the exploration to be updated.
+        change_list: list(dict). List of dicts, each representing a _Change 
+            object. These changes are applied in sequence to produce the resulting
+            exploration.
+        commit_message: str or None.  A description of changes made to the state.
```

Comment:
No -- str is OK. These are Python types you're dealing with here, not datastore types.